stdlib/public/TensorFlow/ShapedArray.swift

```diff
@@ -345,26 +345,38 @@ internal extension ShapedArray where Scalar : _TensorFlowDataTypeCompatible {
 }
 
 public extension ShapedArray {
+  /// The number of dimensions of the `ShapedArray`.
```

Comment:
I haven't seen Swift API docs use a definite article followed by the type name to denote `self`. `Array`'s [`count`](https://developer.apple.com/documentation/swift/array/2943906-count) says "in the array". So perhaps we should do the same.

Reply:
You can make a decision. If it's to be changed, other places should follow as well.

Reply:
I chose to use "array" for now. "Shaped array" might be good too, to distinguish from normal `Array` and usages of "array".

Done in https://github.com/apple/swift/pull/23093/commits/7b40a3e955845cb5b670e230dea499b91c83f895.